cogs/misc.py

```diff
@@ -1,178 +1,197 @@
+    async def clear_memos_db(self, ctx):
+        await ctx.send(':warning: Clearing DB!')
+        
+        dbcur = ctx.bot.database.cursor()
+        dbcur.execute('''DROP TABLE memos''')
+        dbcur.execute('''CREATE TABLE memos(memo TEXT, user_id INTEGER, length INTEGER, start_time INTEGER)''')
+        
+        dbcur.close()
+        ctx.bot.database.commit()
+        
+        await ctx.send('Done!')
+    
+    @commands.command()
+    async def roll(self, ctx, sides: int, how_many_dice: int = 1):
+        '''Rolls a dice.'''
+        if sides < 1 or how_many_dice < 1:
+            return await ctx.send('Yeah, sorry, but you can\'t roll something that doesn\'t exist.')
+
+        if sides > 100 and how_many_dice > 1:
+            return await ctx.send('Woahh, that\'s a lot of sides. Try lowering it below 100?')
+        elif how_many_dice > 30:
+            return await ctx.send('Woahh, that\'s a lot of dice to roll. Try lowering it below 30?')
+
+        if sides == 1:
+            return await ctx.send('All {} of the 1-sided dice shocking rolled a 1.'.format(how_many_dice))
+
+        # easter eggs
+        if sides == 666:
+            return await ctx.send('Satan rolled a nice {} for you.'.format(random.randint(1, sides)))
+        elif sides == 1337:
+            return await ctx.send('Th3 {}-51d3d d13 r0ll3d 4 {}.'.format(sides, random.randint(1, sides)))
+        elif isinstance(ctx.channel, discord.abc.GuildChannel) and sides == ctx.guild.member_count:
+            members = sorted(ctx.guild.members, key=lambda m: m.joined_at)
+            rolled = random.randint(0, len(members) - 1)
+            chosen = members[rolled]
+            return await ctx.send('{}? That\'s how many users are on the server! Well, your die rolled a {}, '
+                                  'and according to the cache, that member is `{}`.'.format(sides, rolled + 1,
+                                                                                            chosen.name))
+        result = ', '.join(str(random.randint(1, sides)) for _ in range(how_many_dice))
+
+        die_message = 'The {} {}-sided {} rolled {}.'.format(
+            how_many_dice, sides, 'die' if how_many_dice == 1 else 'dice', result)
+
+        # :blobrolleyes: wooningc
+        if len(die_message) > 2000:
+            return await ctx.send("Congratulations, you've managed to roll a die that we can't send.")
+
+        await ctx.send(die_message)  
+
+    @commands.command()
+    async def isprime(self, ctx, num: int):
+        """Very simple prime number checker. Limit is 1000000"""
+        if num in [2, 3, 5, 7]:
+            return await ctx.send("*sigh* Yes, {} is prime.".format(num))
+        if num % 2 == 0:
+            return await ctx.send('Why would you think {}, an even number, is prime!?'.format(num))
+        if num % 5 == 0: 
+            return await ctx.send('Composite. {} literally ends in a 5...'.format(num))
+        if num >= 1000000:

```

Comment:
A _bit_ pedantic, but `>=` is redundant here as `==` is caught by `num % 5` earlier in the code. (Just use `>`)